ssl/ssl_rsa.c

```diff
@@ -330,7 +323,10 @@ int SSL_CTX_use_certificate_file(SSL_CTX *ctx, const char *file, int type)
         j = ERR_R_PEM_LIB;
         cert = PEM_read_bio_X509(in, &x, ctx->default_passwd_callback,
                                  ctx->default_passwd_callback_userdata);
-    }
+    }else {
+		ERR_raise(ERR_LIB_SSL, SSL_R_BAD_SSL_FILETYPE);
+		goto end;
+	}
```

Comment:
Please replace tabs with 4 spaces. Also missing space before `else`.

Also this is getting borderline on CLA: trivial. Could you please consider submitting a regular CLA?

Reply:
Commit done. And I sent signed ICLA to email.